src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -0,0 +1,11 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class ColorSupplier {
+    String getRandomColor() {
+        int colorIndex = new Random().nextInt(Color.values().length);
```

Comment:
Don't create new Random every time this method is invoked. Create Random as a class field
```suggestion
        int colorIndex = random.nextInt(Color.values().length);
```